integration-tests/src/test/resources/queries/wikipedia_editstream_queries.json

```diff
@@ -373,7 +373,7 @@
             "context": {
                 "useCache": "true",
                 "populateCache": "true",
-                "timeout": 60000
+                "timeout": 180000
```

Comment:
I remember that conversation and still stand by that assessment. The clarification here for posterity is appreciated :)